### PR TITLE
Added environment variable to affect Elasticsearch pagination size

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
#### What are the relevant tickets?
- fixes https://github.com/mitodl/micromasters/issues/1596

#### config:
set ``ELASTICSEARCH_DEFAULT_PAGE_SIZE=10`` in .env file.

#### Any background context you want to provide?
- put elastic search page size in env variable. So that we can user same data across app and tests.
- fixed minor UI issue on search header. 

@pdpinch @aliceriot @noisecapella 

#### Screenshots (if appropriate)
screen shot 2016-11-11 at 3 23 08 pm

### CODE REVIEW

**Comment by noisecapella** [comment]
What part of this PR applies the page size setting to Elasticsearch?

**Comment by noisecapella** [comment]
Sorry, to be more specific, can you change the back end to add the size from the environment variable, so that unit tests can easily change this value?

**Comment by amir-qayyum-khan** [comment]
ok

**Comment by amir-qayyum-khan** [comment]
Done @noisecapella

File: search/api.py [hunk 1]
```diff
@@ -72,6 +72,7 @@ def create_search_obj(user, search_param_dict=None):
     # the following filter should come first because the sequence matters in applying them
     search_obj = search_obj.filter(create_program_limit_query(user))
     if search_param_dict is not None:
+        search_param_dict['size'] = settings.DEFAULT_ES_PAGE_SIZE
```
Comment (noisecapella) [inline search/api.py]: What if `search_param_dict` is `None`? We still need to insert the size in that case

File: micromasters/settings.py [hunk 2]
```diff
@@ -487,6 +487,7 @@ def get_var(name, default):
 CELERY_TIMEZONE = 'UTC'
 
 # Elasticsearch
+DEFAULT_ES_PAGE_SIZE = get_var('DEFAULT_ES_PAGE_SIZE', 10)
```
Comment (noisecapella) [inline micromasters/settings.py]: Can you rename this to `ELASTICSEARCH_DEFAULT_PAGE_SIZE` to be consistent with the other `ELASTICSEARCH` environment variables?

**Comment by noisecapella** [comment]
Can you use `override_settings` on the classes `SearchTests` in `search/views_test.py` and `SearchAPITests` in `search/api_test.py`? We should set a high value for that environment variable so that all search results comes through in the first page

**Pull request merged**